app/templates/rentals.hbs

```diff
@@ -0,0 +1,8 @@
+<h2>Look up information for a specific rental</h2>
+
+{{outlet}}
+
+<footer>
```

Comment:
See the index route. we are no longer using a footer, but rather a top banner.  See the current index.hbs